src/__tests__/users.int.test.ts

```diff
@@ -64,6 +70,40 @@ describe('/users', () => {
 			});
 		});
 
+		it('returns the user with source if a sourceId has been assigned', async () => {
+			await createOrUpdateFunder({
+				shortCode: 'corpCorp',
+				name: 'CorpCorp',
+			});
+			const source = await createSource({
+				funderShortCode: 'corpCorp',
+				label: 'CorpCorp',
+			});
+
+			await createUser({
+				authenticationId: 'totallyDifferentUser@example.com',
+				sourceId: source.id,
+			});
+			const { count: userCount } = await loadTableMetrics('users');
+
+			const response = await request(app)
+				.get('/users?authenticationId=totallyDifferentUser@example.com')
+				.set(authHeaderWithAdminRole)
+				.expect(200);
+			expect(response.body).toEqual({
+				total: userCount,
+				entries: [
+					{
+						authenticationId: 'totallyDifferentUser@example.com',
+						createdAt: expectTimestamp,
+						id: 3,
+						source,
```

Comment:
Thoughts on this approach (of using the `source` object as returned by `createSource` rather than explicitly defining the shape of the expected result)?